CoffeeShopUI_Angular/src/app/components/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,59 @@
+<div style="display: flex; justify-content: center; margin: 7% 0px;">
+    <div class="box">
+        <h1>Sign in</h1>
+        <div>
+            <form [formGroup]="signInForm" (ngSubmit)="onSubmit()">
+                <div class="fields">
+                    <div class="icon">
+                  <svg width="16" height="12" viewBox="0 0 16 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Don't forget to look into some ways of using svg's without pasting them in the html of the component.